frontend/src/views/FAQPage.vue

```diff
@@ -62,7 +62,7 @@ export default defineComponent({
       />
     </div>
     <div class="fr-col-lg-4 fr-col-xl-4">
-      <ImagePresentation :src="image.faqSvg" alt="Illustration de la page de la FAQ" />
+      <ImagePresentation :src="image.faqSvg" alt="" />
```

Comment:
On peut complétement supprimer l'attribut. Dans le composant ImagePresentation le alt n'est pas obligatoire et a une valeur par défaut à ''.

```suggestion
      <ImagePresentation :src="image.faqSvg" alt="" />
```
```suggestion
      <ImagePresentation :src="image.faqSvg" />
```